### PR TITLE
empty giftcard added

### PR DESCRIPTION
**Pull request by krunalraut**
new giftcard commit

### CODE REVIEW

**Comment by arameshkumar02** [comment]
i can see the giftcard class

**Comment by PurviShah06** [comment]
keep it up good work

File: src/main/java/org/example/GiftCardPage.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package org.example;
+
+public class GiftCardPage {
+}
```
Comment (NehaPatel9) [inline src/main/java/org/example/GiftCardPage.java:4]: Please add some comments to the class

Comment (krunalraut) [reply src/main/java/org/example/GiftCardPage.java:4]: ok

**Pull request merged**